cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -79,7 +80,6 @@
     public static final String CRUISE_CONTROL_HARD_GOALS_CONFIG_KEY = "hard.goals";
```

Comment:
Even if you define the list in this class you can still move the key to the Enum and just leave the default value as `null`?

Reply:
I can but it looked a bit weird to my eyes, as it has no value and it is just a config key without value :/

Reply:
I added it there, with making default value as empty string, as all those default values are joined list of String goals.
Would you prefer null instead as default value?

Reply:
I still think it feel wrong to have the CC defaults defined in 2 places. I think you should move the goal list definitions into the Enum too (as that is where all the other definitions are made, but then that makes the class a bit messy. WDYT @ppatierno?